### DISCUSSION TITLE
Meshcentral does not start.

### DISCUSSION THREAD

**Discussion by illatus** [Ylianst/MeshCentral#5364] [Q&A]
Hello again,

I created the service needed in order to make the mesh central automatically start. After that my mesh central went black.
i tried to reboot my ubuntu machine it didnt work. I tried to manually start with the command `node node_module/meshcentral`
but it tries to install ua-parser-js@1.0.35 with an error : 
![Screenshot_1](https://github.com/Ylianst/MeshCentral/assets/35657321/a3e6420a-e9be-43f4-b009-7de7db591823)

Any idea why is not working ? I have also checked the meshcentral.service file with the configuration. Everything seems right.

**Comment by si458** [comment]
weird? someone else is having the same issue with ua-parser-js?
https://github.com/Ylianst/MeshCentral/issues/5329#issuecomment-1727364254
what is the OS you are running?
have you tried stopping meshcentral in the background `sudo systemctl stop meshcentral.service` then running it manually with
`sudo node node_modules/meshcentral` ?

EDIT: i would also create a folder called `meshcentral` inside your home directory, then go into it `cd meshcentral` then run all install commands etc from that folder, so it doesnt get confused

**Reply by illatus** [reply]
hey,

Thank you for your reply. I am running ubuntu 22.04 the newest version. I tried to stop it and run it manually but it didnt work. I am still getting the same error.

**Comment by illatus** [comment]
Anyone any suggestion ?

**Comment by Nothing2Do** [comment]
I suggest, a permission problem ... "sudo chown -R $USER:$USER ~/meshcentral-data/" (or any meshcentral directory). Perhaps, the npm directory too (the same with ~/node_modules). you should have a bad username in your "meshcentral.service"
